pokemon-app/src/components/Search.js

```diff
@@ -13,65 +13,70 @@ const Search = ({ getPokeByNameOrIdAPI, isEnemy, setIsEnemy }) => {
 
   const onSubmit = (e) => {
     e.preventDefault();
-
     const pokeLowerCase = pokemonSearchValue.toLowerCase();
 
     setHasSelected(true);
     getPokeByNameOrIdAPI(pokeLowerCase);
   };
+  const removePokemon = () => {
+    if(enemyPokemon === null){
+      if(pokemon !== null){
```

Comment:
You can also shorthand this as `if(!enemyPokemon)` and `if(pokemon)`, as `null` is falsy.